Quick/World.swift

```diff
@@ -57,12 +57,7 @@ final internal class World: NSObject {
     // MARK: Singleton Constructor
 
     private override init() {}
-    private struct Shared {
-        static let instance = World()
-    }
-    internal class func sharedWorld() -> World {
-        return Shared.instance
-    }
+    static let sharedWorld = World()
```

Comment:
Very nice, thanks! :+1:

You could tell I wasn't super knowledgeable about globals in Swift when I wrote this... :stuck_out_tongue: